### ISSUE TITLE
ubuntu 17.10 problem library problem with 1.11 latest release

### ISSUE THREAD

**Issue by petabytestorage** [jessevdk/cldoc#132]
[scplat] ./support/generate-cldocs                             add-google-auth 
shell script tries to make docs like
```
#!/usr/bin/zsh
cd $(dirname $0)
cd ..
SOURCES=(**/*.*pp)
#SOURCES=(**/mv1db.hpp)
cldoc generate -Isrc/sop/include -Isrc/pconfig/include -Isrc/everynet/include -Isrc/scjwt/include -Isrc/sjson/include -Isrc/mv1db/include -Isrc/odb-auto/include -Isrc/dbcore/include -Isrc/stepcore/include -Isrc/bz1db/include -Isrc/cryptik/include -std=c++14 -Isrc/echobridge/include -Isrc/sview/include -Isrc/ezreq/include -Isrc/util/include -Isrc/fitbitnet/include -Isrc/sbase64/include -Isrc/replay-manager/include -Isrc/scatch/include -- --output=html -- $SOURCES
```

Traceback (most recent call last):
  File "/usr/local/bin/cldoc", line 11, in <module>
    load_entry_point('cldoc==1.11', 'console_scripts', 'cldoc')()
  File "/usr/local/lib/python2.7/dist-packages/cldoc-1.11-py2.7.egg/cldoc/__init__.py", line 57, in run
    run_generate(rest)
  File "/usr/local/lib/python2.7/dist-packages/cldoc-1.11-py2.7.egg/cldoc/__init__.py", line 27, in run_generate
    cmdgenerate.run(args)
  File "/usr/local/lib/python2.7/dist-packages/cldoc-1.11-py2.7.egg/cldoc/cmdgenerate.py", line 125, in run
    t.process()
  File "/usr/local/lib/python2.7/dist-packages/cldoc-1.11-py2.7.egg/cldoc/tree.py", line 166, in process
    self.index = cindex.Index.create()
  File "/usr/local/lib/python2.7/dist-packages/cldoc-1.11-py2.7.egg/cldoc/clang/cindex.py", line 2696, in create
    return Index(conf.lib.clang_createIndex(excludeDecls, 0))
  File "/usr/local/lib/python2.7/dist-packages/cldoc-1.11-py2.7.egg/cldoc/clang/cindex.py", line 210, in __get__
    value = self.wrapped(instance)
  File "/usr/local/lib/python2.7/dist-packages/cldoc-1.11-py2.7.egg/cldoc/clang/cindex.py", line 4139, in lib
    register_functions(lib, not Config.compatibility_check)
  File "/usr/local/lib/python2.7/dist-packages/cldoc-1.11-py2.7.egg/cldoc/clang/cindex.py", line 4086, in register_functions
    register(f)
  File "/usr/local/lib/python2.7/dist-packages/cldoc-1.11-py2.7.egg/cldoc/clang/cindex.py", line 4083, in register
    return register_function(lib, item, ignore_errors)
  File "/usr/local/lib/python2.7/dist-packages/cldoc-1.11-py2.7.egg/cldoc/clang/cindex.py", line 4064, in register_function
    raise LibclangError(msg)
cldoc.clang.cindex.LibclangError: /usr/lib/x86_64-linux-gnu/libclang-4.0.so.1: undefined symbol: clang_EnumDecl_isScoped. Please ensure that your python bindings are compatible with your libclang.so version.
[scplat] sudo apt install libclang-dev                         add-google-auth 
Reading package lists... Done
Building dependency tree       
---
As a workaround I tried to run the cldoc-static.js instead and it gave a different error about jsdom not being installed.  The fork I mentioned here combined with the symlink trick fixed the earlier version of cldoc but I think the "clang_EnumDecl_isScoped" missing symbol is new. Seems like a clang related problem but so far I have only seen it when trying to use cldoc so I thought I would post it here to see if you might have any advice.

**Comment by jessevdk** [comment]
I had to upgrade the clang bindings to make them compatible with python3, but to do so I had to take them from clang 6. I've now made the bindings compatible again with 3.9 (released as 1.11.1) so it should work for you. Can you verify?

**Comment by petabytestorage** [comment]
thanks that seems to have fixed the clang binding symbol issue. now on to a new bug that i will open another issue for.
